conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -41,6 +42,12 @@ class SoundscapeAttr(Enum):
     time = auto()
     NN = auto()
     N0 = auto()
+    A_Den = auto()
```

Comment:
doesn't the value need to be manually encoded as "A.Den"?

Reply:
done